src/components/common-session-component/navbar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const Navbar = () => {
+  return (
+    <div>
+      <nav className="navbar">
+        <Link to="/transaction" className="link">
+          Transactions
+        </Link>
+        <Link to="/deposit" className="link">
+          Deposit
+        </Link>
+        <Link to="/withdrawal" className="link">
+          Withdraw
+        </Link>
+        <Link to="/transfer" className="link">
+          Transfer
+        </Link>
+        <Link to="/account-profile" className="link">
+          Account Profile
+        </Link>
+        <Link to="/logout" className="link">
```

Comment:
There is no page called logout and there shouldn't be. You should attach an onClick function to the Link that will correctly log the user out and the redirect to login page.
Something like:
```
<Link className="link" onClick={handleLogout}>
     Log out
</Link>
```